src/lib/loadInstructions.js

```diff
@@ -137,6 +267,34 @@ module.exports = function loadInstructions() {
         }
       }
 
+      const totalBits = doc.encoding?.match?.length === 16 ? 16 : 32;
+      const filledFields = expandBitfieldFields(fields, totalBits);
+      const bitfieldJSON = {
+        reg: filledFields.map(f => {
+          let name = f.label;
+          // Constant fields have their label part before "=" removed (e.g., "funct7=0000000" → "0000000").
+          if (f.kind === "const" && typeof name === "string") {
```

Comment:
Under what circumstances would the condition `typeof name === "string"` be false? I hope it is the case that we always have a name, and that the name is always a string. If not, it would be better to crash with an error than to keep on truckin'.